addons/counterMortar/functions/fnc_selectResponse.sqf

```diff
@@ -0,0 +1,48 @@
+#include "script_component.hpp"
+/*
+    Author:
+        Bridg
+
+    Description:
+        selects response between air units, ground units and counter battery
+
+    Parameters:
+        0: _counterBatteryUnit <OBJECT>
+
+    Return value:
+        Nothing
+*/
+
+params ["_counterBatteryUnit"];
+
+if (count (GVAR(groundVehicleTypes) + GVAR(airVehicleTypes) + GVAR(counterBatteryUnits)) == 0) exitWith {diag_log "UKSF: ANTI MORTAR --- No counter units defined, check preInit ---"};
+// probably needs some kind of removal of the class EH to prevent spam
+
+// private _forceChance = 4; // for debug
+private _forceChance = ceil(random 10);
```

Comment:
Why `ceil`?
The `2.5` comparison below is pointless as the value can only be 3 if a decimal greater than 2.
Either don't use ceil, or compare against 2 or 3